go.mod

```diff
@@ -1,4 +1,6 @@
-go 1.21
+go 1.23
```

Comment:
Oh that's problematic, we need to downgrade x/tx to go 1.21 then (and in CI too)